install_tools_wrapper.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
Why is this file needed again?

Reply:
I plan to remove it when this PR becomes green
The -v option was also added in https://github.com/bgruening/docker-galaxy-stable/pull/403/files, but I was just too lazy to patch the 17.09 image too + update the bgruening/galaxy-sequence-tools:17.09 image